match_vr_halos.py

```diff
@@ -260,6 +260,61 @@ def consistent_match(match_index_12, match_index_21):
     # If we retrieved our own halo index, we have a match
     return np.where(match_back == local_halo_index, 1, 0)
 
+def get_match_vr_halos_args(comm):
```

Comment:
We'll need to remove the import of the removed command_line_args file and change the call to get_match_vr_halos_args() to reference this version instead of the deleted file.